admin_manual/release_notes/upgrade_to_28.rst

```diff
@@ -18,6 +18,27 @@ Web server configuration
   * For Nginx, please refer to our recommended :ref:`Nginx configuration<nginx-config>`.
   * For other setups, make sure to add ``.mjs`` to the list of static file extensions in web server configs and in case define its MIME type in ``/etc/mime.types``.
 
+Setup Checks
+------------
+
+The setup checks (the ones visible under *Administration settings->Overview*) that previously ran from the web browser now run server-side rather than from the browser. 
```

Comment:
This is new in NC 28? Or was the change even older?

Reply:
Changed for 28